### PR TITLE
INTPYTHON-348 Update raw_query tests for raw_mql

### PR DESCRIPTION
**Pull request by aclark4life**
feature: https://github.com/mongodb-labs/django-mongodb/pull/183

### CODE REVIEW

**Comment by timgraham** [comment]
Looks to be on the right track.

File: tests/raw_query/models.py [hunk 1]
```diff
@@ -1,8 +1,10 @@
 from django.db import models
 from django_mongodb.manager import MongoManager
+from django_mongodb.fields import ObjectIdAutoField
 
 
 class Author(models.Model):
+    id = ObjectIdAutoField(primary_key=True)
```
Comment (timgraham) [inline tests/raw_query/models.py]: You don't need to add an explicit primary key to all these methods. Put `DEFAULT_AUTO_FIELD = "django_mongodb.fields.ObjectIdAutoField"` in your test settings file.

File: tests/raw_query/tests.py [hunk 2]
```diff
@@ -247,46 +248,46 @@ def test_query_representation(self):
         """
         Test representation of raw query with parameters
         """
-        query = "SELECT * FROM raw_query_author WHERE last_name = %(last)s"
-        qset = Author.objects.raw(query, {"last": "foo"})
+        query = [{"$match": {"last_name": "%(last)s" % {"last": "foo"}}}]
```
Comment (timgraham) [inline tests/raw_query/tests.py:251]: This test seems useless if there is no need for a `params` argument to `raw_mql()`. Using string interpolation to construct the query is not adding any value, so you might just skip test like these if they aren't applicable.

Comment (aclark4life) [reply tests/raw_query/tests.py:251]: I'm not sure I understand why that would be needed with `MQL` or `SQL`. There is another test that includes `params` kwarg (are we planning to support this?) This one seems to be testing string interpolation for no reason that I understand …

Comment (timgraham) [reply tests/raw_query/tests.py:251]: Query parameters are passed separately in SQL-land [to protect against SQL injection attacks](https://docs.djangoproject.com/en/5.1/topics/security/#sql-injection-protection).

**Comment by aclark4life** [comment]
Superseded by #12

**Pull request closed without merging**